content/projects/gsoc/2024/project-ideas/fixing-backend-extension-indexer-tool.adoc

```diff
@@ -17,7 +17,7 @@ links:
 ---
 
 === Background
-The Backend Extension Indexer Tool currently used to display all plugins at link:https://www.jenkins.io/doc/developer/extensions/[https://www.jenkins.io/doc/developer/extensions/] is not working as expected. The tool is not able to display all the plugins. And this project involves identifying the underlying issues and fixing them. We currently know that if the plugin has been modernised, then they would no longer be fetched by the tool.
+The Backend Extension Indexer Tool currently used to display all plugins at link:https://www.jenkins.io/doc/developer/extensions/[https://www.jenkins.io/doc/developer/extensions/] is not working as expected. The tool is not able to display all the plugins currently. And this project involves identifying the underlying issues and fixing them. We currently know that if the plugin has been modernised, then they would no longer be fetched by the tool. The tool itself can be found on GitHub at link:https://github.com/jenkins-infra/backend-extension-indexer[]. A list of all plugins for further investigation can be found at link:https://plugins.jenkins.io/[].
```

Comment:
```suggestion
The Backend Extension Indexer Tool currently used to display all plugins at link:/doc/developer/extensions/[https://www.jenkins.io/doc/developer/extensions/] is not working as expected. The tool is not able to display all the plugins currently. This project involves identifying the underlying issues and fixing them. We currently know that if the plugin has been modernized, then it would no longer be fetched by the tool. The tool itself can be found on GitHub at link:https://github.com/jenkins-infra/backend-extension-indexer[]. A list of all plugins for further investigation can be found at link:https://plugins.jenkins.io/[].
```